dts/common/nordic/nrf54h20.dtsi

```diff
@@ -168,6 +168,11 @@
 				interrupts = <64 NRF_DEFAULT_IRQ_PRIORITY>,
 					     <65 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			cpuapp_resetinfo: resetinfo@1e000 {
+				compatible = "nordic,nrf-resetinfo";
+				reg = <0x1e000 0x1000>;
```

Comment:
Could you also add entries in `validate_base_addresses.c` for these new nodes?

Reply:
Rebased and added the `validate_base_addresses.c` entries.